src/nrnoc/extcelln.c

```diff
@@ -145,18 +151,21 @@ i_membrane = sav_g * (NODERHS(nd)) + sav_rhs;
 #endif
 }
 
+static int nparm() {
+#if I_MEMBRANE
+  return 3*(nlayer) + 4;
+#else
+  return 3*(nlayer) + 1;
+#endif
```

Comment:
nitpick : a docstring for function or `+4` and `+1` would be helpful?

Reply:
Explained those values in the comments.